### DISCUSSION TITLE
Does the table editor support GENERATED ALWAYS ?

### DISCUSSION THREAD

**Discussion by budditama** [supabase/supabase#4117] [Questions] [answered]
Hello,

I can´t find a way to specify a generated column from the table editor. Is it supported ?

If it is not, can someone show me where can I read how to do it from a command line ?

Thanks.

**Comment by GaryAustin1** [comment, marked as answer]
Use the SQL editor.   
Then something like this (from stack overflow, lots more examples there)
alter table table_name
  add fullname_col text generated always as (firstname_col || lastname_col) stored;

**Reply by budditama** [reply]
Thanks a lot, Gary
